go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -930,7 +938,7 @@ func (be *BuiltinBackupEngine) backupFile(ctx context.Context, params BackupPara
 				// Close gzip to flush it, after that all data is sent to writer.
 				closeCompressorAt := time.Now()
 				params.Logger.Infof("Closing compressor for file: %s %s", fe.Name, retryStr)
-				if cerr := closer.Close(); err != nil {
+				if cerr := closeWithRetry(ctx, params.Logger, closer, "compressor"); err != nil {
```

Comment:
It looks like this bug was here before as well, but this should be checking `cerr` and not `err`, right?